README.md

```diff
@@ -29,15 +29,29 @@ Also:
 
 Like others, Smithay as a compositor library has its own sample compositor: anvil.
 
-You can run it with cargo after having cloned this repository:
-
+You can run it with cargo after having cloned this repository and installed the required dependencies:
+
+- `libdbus-glib-1`
+- `libudev`
+- `libsystemd`
+- `libxkbcommon`
+- `libinput`
+- `libgbm`
+
+```bash
```

Comment:
```suggestion
```bash
If you enable X11 support:
- `xwayland`
If you enable libseat (seatd, logind, elogind) support:
- [libseat](https://git.sr.ht/~kennylevinsen/seatd)
```

Reply:
Wops, I meant for this to be above bash block, by bad